File: String Handling/lexicographicalSort.py
def sort(arr):
    ret = arr

    for i in range(len(ret)-1):
        for j in range(len(ret)-i-1):
            if ret[j] > ret[j+1]:
                temp = ret[j]
                ret[j] = ret[j+1]
                ret[j+1] = temp

    return ret

arr = ["hello","world","qux","foo","bar"]
print(sort(arr))

File: String Handling/lexicographicalSort.rs
fn sort(arr:Vec<&str>)->Vec<&str>{
    let mut ret = arr;

    for i in 0..ret.len()-1{
        for j in 0..ret.len()-i-1{
            if ret[j] > ret[j+1]{
                let temp = ret[j];
                ret[j] = ret[j+1];
                ret[j+1] = temp;
            }
        }
    }
    ret
}
fn main(){
    let arr:Vec<&str> = ["hello","world","qux","foo","bar"].to_vec();
    println!("{:?}",sort(arr))
}